ui/mirage/scenarios/default.js

```diff
@@ -44,14 +46,74 @@ function smallCluster(server) {
   server.create('feature', { name: 'Dynamic Application Sizing' });
   server.createList('agent', 3, 'withConsulLink', 'withVaultLink');
   server.createList('node', 5);
-  server.createList('job', 5, { createRecommendations: true });
+  server.createList('job', 1, { createRecommendations: true });
   server.createList('allocFile', 5);
   server.create('allocFile', 'dir', { depth: 2 });
   server.createList('csi-plugin', 2);
 
+  // #region evaluations
+
+  // Branching: a single eval that relates to N-1 mutually-unrelated evals
+  const NUM_BRANCHING_EVALUATIONS = 3;
+  Array(NUM_BRANCHING_EVALUATIONS)
+    .fill()
+    .map((_, i) => {
+      return {
+        evaluation: server.create('evaluation', {
+          id: `branching_${i}`,
+          previousEval: i > 0 ? `branching_0` : '',
+          jobID: pickOne(server.db.jobs).id,
+        }),
+
+        evaluationStub: server.create('evaluation-stub', {
+          id: `branching_${i}`,
+          previousEval: i > 0 ? `branching_0` : '',
+          status: 'failed',
+        }),
+      };
+    })
+    .map((x, i, all) => {
+      x.evaluation.update({
+        relatedEvals:
+          i === 0
+            ? all.filter((_, j) => j !== 0).map((e) => e.evaluation)
+            : all.filter((_, j) => j !== i).map((e) => e.evaluation),
+      });
+      return x;
+    });
```

Comment:
`question` why aren't we able to use the `createList` function to generate evaluations here and down below?

Should we encapsulate this logic into a helper with some documentation since it breaks the pattern of this function?